### PR TITLE
feat(ui-mode): add key shortcuts for playwright uI test runner

### LINKED ISSUE #28899: [Feature] - Key Shortcuts for Playwright UI Test Runner
**Summary**:
Introduce keyboard shortcuts for common actions in the Playwright UI test runner to enhance developer productivity and streamline the test execution process.

**Feature Request**:
As a user of Playwright, I would like the ability to use keyboard shortcuts to perform the following actions within the UI test runner:

1. **Stop Current Test Run**: A shortcut to immediately stop the execution of the currently running test(s).
2. **Run Tests**: A shortcut to start running tests without the need to click through the UI.
3. **Refresh Tests**: A shortcut to refresh the list of tests or the test state, useful for when tests have been added or modified.

**Proposed Shortcuts**:
- Stop Current Test Run: `Ctrl/Cmd + S`
- Run Tests: `Ctrl/Cmd + R`
- Refresh Tests: `Ctrl/Cmd + U`

**Benefits**:
- **Increased Productivity**: Users can manage their test executions quicker and more efficiently.
- **Enhanced Usability**: Reduces the reliance on mouse navigation, streamlining the workflow.
- **Consistency**: Brings Playwright's UI in line with other development tools that utilize keyboard shortcuts for common actions.

### PR DESCRIPTION
**Pull request by dev-jonghoonpark**
related issue: #28899 

## Motivation: 

Introduce keyboard shortcuts for common actions in the Playwright UI test runner to enhance developer productivity and streamline the test execution process.

## Modification:

Make shortcut keydown event in `uiModeView.tsx`

## Result:

- add three shortcut in ui test runner
  - Run Tests: Ctrl/Cmd + R
  - Stop Tests: Ctrl/Cmd + S
  - Refresh Tests: Ctrl/Cmd + U
- Close #28899

### CODE REVIEW

File: packages/trace-viewer/src/ui/uiModeView.tsx [hunk 1]
```diff
@@ -177,6 +177,27 @@ export const UIModeView: React.FC<{}> = ({
...
+    const onShortcutEvent = (e: KeyboardEvent) => {
+      if (e.code === 'KeyR' && (e.metaKey || e.ctrlKey)){
+        e.preventDefault();
+        runTests('bounce-if-busy', visibleTestIds);
+      } else if (e.code === 'KeyS' && (e.metaKey || e.ctrlKey)) {
+        e.preventDefault();
+        sendMessageNoReply('stop');
+      } else if (e.code === 'KeyU' && (e.metaKey || e.ctrlKey)) {
+        e.preventDefault();
+        reloadTests();
```
Comment (mxschmitt) [inline packages/trace-viewer/src/ui/uiModeView.tsx:190]: I feel like KeyR should be reload which seems more natural to the users. Maybe Enter+Shift should be run?

**Comment by dev-jonghoonpark** [comment]
As comments are edited
Finally, I changed it to 'Ctrl/Cmd + Enter'.
@mxschmitt

**Comment by mxschmitt** [comment]
Sorry for the late reply. We thought a little bit about it, when you are on a test item, you can already as of today press <kbd>Enter</kbd> in order to run a test. So I'd start with F5 / F6 now for reloading and stopping the current tests.

What do you think? I pushed to your branch, but need to adjust tests.

**Comment by dev-jonghoonpark** [comment]
F5 / F6 are also good.
I adjusted the tests.

**Pull request merged**